src/main/resources/assets/sodium-extra/lang/en_us.json

```diff
@@ -279,8 +280,8 @@
   "sodium-extra.option.toasts.tooltip": "If enabled, advancement and crafting recipe popups will be displayed.",
   "sodium-extra.option.tutorial_toast": "Tutorial Toasts",
   "sodium-extra.option.tutorial_toast.tooltip": "If enabled, tutorial popups will be displayed.",
-  "sodium-extra.option.use_adaptive_sync.name": "Adaptive",
-  "sodium-extra.option.use_adaptive_sync.tooltip": "If enabled, V-Sync will be able to swap mid-frame and disable itself at times, usually notably helping responsiveness.",
+  "sodium-extra.option.use_adaptive_sync.name": "Adaptive Sync",
```

Comment:
This isn't FreeSync or G-Sync, it can not be called Adaptive Sync. It's technically Adaptive VSync.

Reply:
Fixing that part then.

Reply:
I left it as `Adaptive` because it will display VSync as adaptive mode.
![image](https://user-images.githubusercontent.com/30311066/201174106-69e67a1b-e9e0-4e90-9958-975de2fef4d2.png)

Reply:
![image](https://user-images.githubusercontent.com/30311066/201174582-686d9456-f3e9-4858-8478-fda146077b34.png)

Reply:
Never thought of this then, oops.